.github/workflows/builds.yml

```diff
@@ -175,3 +175,31 @@ jobs:
       run: |
         source ${GITHUB_WORKSPACE}/.github/ci_setup.sh ${{ matrix.PLATFORM.NAME }}
         cmake --build build
+
+  benchmark-build:
```

Comment:
Should this stay here or is it just for testing?

Reply:
I'm thinking we could leave this here to quickly diagnose the benchmark build, but I'm happy to go either way.

Reply:
OK, should we turn off the benchmark build on the other (ubuntu) images then?

Reply:
Hmm, I wouldn't worry too much about that; the whole CI job takes 3 minutes; most of the time is spent on testing.

Reply:
Fair point